deploy/002_caller_only_swapper.ts

```diff
@@ -10,9 +10,9 @@ const deployFunction: DeployFunction = async function (hre: HardhatRuntimeEnviro
 
   await deployThroughDeterministicFactory({
     deployer,
-    name: 'DCAHubSwapper',
-    salt: 'MF-DCAV2-DCAHubSwapper-V1',
-    contract: 'contracts/DCAHubSwapper/DCAHubSwapper.sol:DCAHubSwapper',
+    name: 'DCAHubSwapper', // We will use the old name to avoid re-deploying
```

Comment:
Why would we not re-deploy this one? Idea is to reuse the old one for swapping for caller?

Reply:
Yeah, that's the plan. Now that you say it, we should use the same salt so that the contracts have the same address. Since the code is the same for 'swap for caller', then we can re-use the currently deployed versions
What do you think?

Reply:
Oh I see, so basically maintaining the address, interface and compatibility, but just stripping out everything from the contract.  I mean, I do understand the "why" - but for the sake of consistency I'd change the salt, the name and re-deploy (everything is cheap af, except for Ethereum that it will not be *that* expensive). But that is just who I am and my small OCD with deployment addresses, and consistency all around.

Reply:
Ok, will do! 👍